events/ready.js

```diff
@@ -1,7 +1,15 @@
 const chainMessageAction = require('../eventActions/checkChainMessage');
+const config = require('../config.json');
+const { sequelize } = require('../databaseFiles/connect');
+const remind = require('../commands/remind');
+const tosRemind = require('../eventActions/tosReminderAction.js');
```

Comment:
A bit of a nitpick but most of the `require` statements here don't have the `.js` file extension whereas the `tosRemind` does, maybe remove the extension from all of them or add it for all of them.

Reply:
I standardized them here, but I'm hoping to add this to ESLint at some point.